### PR TITLE
fix: emojis are not visible on iOS Chrome

### PR DESCRIPTION
**Pull request by RealGoodProgrammer**
Fix for hidden emojis selector on Chrome (iOS
image (8)

### CODE REVIEW

**Comment by bludnic** [comment]
@RealGoodProgrammer Attach the Trello card to the PR

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "private": true,
   "scripts": {
     "serve": "vite preview --config vite-pwa.config.ts",
-    "dev": "vite --config vite-pwa.config.ts",
+    "dev": "vite --host 0.0.0.0 --config vite-pwa.config.ts",
```
Comment (bludnic) [inline package.json:10]: Why is this needed?

File: src/components/EmojiPicker.vue [hunk 2]
```diff
@@ -68,6 +68,8 @@ export default defineComponent({
 
 .emoji-picker {
   border-radius: 8px;
+  position: absolute;
+  bottom: 0;
```
Comment (bludnic) [inline src/components/EmojiPicker.vue:72]: Does `<v-menu location="top"` works without these styles?

**Review by bludnic — CHANGES_REQUESTED** [review]

**Review by bludnic — APPROVED** [review]

File: src/components/EmojiPicker.vue (hunk 2 above)
Comment (RealGoodProgrammer) [reply src/components/EmojiPicker.vue:73]: Not in Google Chrome on iOS

**Review by bludnic — APPROVED** [review]

**Comment by bludnic** [comment]
[UI: Add emoji and move attach to the input field (copied from iOS)](https://trello.com/c/x5HC9Uyj/426-ui-add-emoji-and-move-attach-to-the-input-field-copied-from-ios)

**Comment by PaulDremanovich** [comment]
Why do you have the positioning property implemented if you keep hardcoding it early? Perhaps you just need to change the style?

**Comment by RealGoodProgrammer** [comment]
> Why do you have the positioning property implemented if you keep hardcoding it early? Perhaps you just need to change the style?

For the case when it should be static.  Look at the condition. The class will be applied only when the 'absolute' value is passed to it in the property

**Review by PaulDremanovich — APPROVED** [review]
